docs/en-us/dev/SPI.md

```diff
@@ -116,7 +116,7 @@ public class RaceCarMaker implements CarMaker {
 }
 ```
 
-when`ExtensionLoader` loading `CarMaker`  implementation `RaceCar` ，`setWheelMaker`  needs paramType `WheelMaker`  which is also an SPI, It will be automatically loaded .
+when`ExtensionLoader` loading `CarMaker`  implementation `RaceCarMaker` ，`setWheelMaker`  needs paramType `WheelMaker`  which is also an SPI, It will be automatically loaded .
```

Comment:
```suggestion
when`ExtensionLoader` loads `CarMaker`'s  implementation `RaceCarMaker` ，the method `setWheelMaker` needs paramType `WheelMaker` which is also a SPI, It will be automatically loaded .
```